main/resource-access.bs

```diff
@@ -95,6 +95,44 @@ Behaviour pertaining to authorization MUST proceed this optional redirect
 
 ## Linked Data ## {#linked-data}
 
+### Storage ### {#storage}
+
+When a servers supports a data pod, it MUST provide one or more storages
+(`pim:Storage`). A storage is the root container for all of its contained
+resources (see [[#resource-containment]]).
+
+When a server supports multiple storages, the URIs MUST be allocated to
+non-overlapping space.
+
+Servers exposing the storage resource MUST advertise by including the HTTP
+`Link` header with `rel="type"` targeting
+`http://www.w3.org/ns/pim/space#Storage` when responding to request URI.
+
+Clients can determine a resource is of type storage by making an HTTP `HEAD`
+or `GET` request on the target URL, and check for the `Link` header with
+`rel="type"` targeting `http://www.w3.org/ns/pim/space#Storage`.
+
+Clients can determine the storage of a resource by moving up the URI path
+hierarchy until the response includes a `Link` header with `rel="type"`
```

Comment:
Do we have a ref for "moving up hierarchy"? We probably have it documented somewhere else already.

Reply:
I was trying to keep it minimal and close to 3986 as opposed to "containment" but that can also work. We have used "traverse" in discussions eg. "traverse up in the containment hierarchy". And you've used "construct the URL of the parent container by stripping it to the closest slash" in the issue of ACL inheritance algorithm for clients. A bit too verbose for my taste. I'm okay to define the behaviour but perhaps we should do it in one go once the other usage is presented?

Reply:
Ah yes, I thought it already was.